tests/CMakeLists.txt

```diff
@@ -96,20 +121,20 @@ set(tests
     "time"
     "lua_simple"
     "lua_script"
+    "test_map"
+    "test_args_near_unmapped"
     "args-simple args_simple check_simple 40 2"
     "args-more args_simple check_simple 40 2 2 2" # Check additional arguments are ignored
     "args-combined args_simple check_combined 400 2 20"
     "args-negative args_simple check_negative -42"
     "args-long-max args_simple check_llong_max 9223372036854775807"
     "args-long-min args_simple check_llong_min -9223372036854775808"
     "args-ulong-max args_simple check_ullong_max 18446744073709551615"
-    # Order matters, as these depend on some files to be copied in earlier tests
-    "test_map"
-    "test_args_near_unmapped"
     )
 
-two_comp_test(test_two_comps test_two_comps-comp1 test_two_comps-comp2)
+#two_comp_test(test_two_comps test_two_comps-comp1 test_two_comps-comp2)
```

Comment:
Should this be commented out?

Reply:
Yes. I started working towards the multi-compartment implementation, before moving to changing the testing infrastructure and deciding to make this PR. Don't think it's worth removing it for this PR, as it's the next planned feature, but I can if you rather keep this PR cleaner.

Reply:
Works for me.